ts/src/gate.ts

```diff
@@ -606,26 +606,6 @@ export default class gate extends Exchange {
                     'SOLANA': 'SOL',
                     'TRC20': 'TRX',
                 },
-                'networksById': {
```

Comment:
Actually this is used by `networkIdToCode` and `networkCodeToId` but yeah this object is filled automatically from the `networks`